packages/elements/tsconfig.json

```diff
@@ -7,8 +7,7 @@
   "references": [
     { "path": "../i18n" },
     { "path": "../media" },
-    { "path": "../patterns" },
-    { "path": "../units" }
```

Comment:
Since the `units` package is still listed as a dependency in `package.json`, we should keep the reference here as well.

Reply:
Yes, that's a mistake.